python_modules/dagster/dagster/core/execution/context/compute.py

```diff
@@ -312,6 +312,15 @@ def selected_output_names(self) -> AbstractSet[str]:
                 selected_outputs.add(output_name)
         return selected_outputs
 
+    def asset_key_for_output(self, output_name: str = "result") -> AssetKey:
+        asset_output_info = self.pipeline_def.asset_layer.asset_info_for_output(
+            node_handle=self.op_handle, output_name=output_name
```

Comment:
Does this include the prefix? I.e. when using:

```
AssetGroup.from_package_module(...).prefixed("my_prefix")
```

Would the name/asset key automatically include this reference?

Reply:
It will include that reference - i.e. "my_prefix" will show up in the returned asset key.